components/admin/requests/additional-questions/Form.tsx

```diff
@@ -1,55 +1,106 @@
 
 type AdditionalQuestionsFormProps = {
-  data: AdditionalInformationFormData;
-  onChange: (additionalQuestions: AdditionalInformationFormData) => void;
+  additionalInformation: AdditionalInformationFormData;
 };
 
-export default function AdditionalQuestionsForm({ data, onChange }: AdditionalQuestionsFormProps) {
+export default function AdditionalQuestionsForm({
+  additionalInformation,
+}: AdditionalQuestionsFormProps) {
   return (
     <>
-      <FormControl as="fieldset" isRequired>
-        <FormLabel>{'Is the applicant using an accessible converted van?'}</FormLabel>
-        <RadioGroup
+      <Box paddingBottom="24px">
+        <RadioGroupField
+          name="additionalInformation.usesAccessibleConvertedVan"
+          label="Is the applicant using an accessible converted van?"
+          required
           value={
-            data.usesAccessibleConvertedVan === null
+            additionalInformation.usesAccessibleConvertedVan === null
               ? undefined
-              : data.usesAccessibleConvertedVan
-              ? '0'
-              : '1'
-          }
-          onChange={value =>
-            onChange({ ...data, usesAccessibleConvertedVan: value === '0' ? true : false })
+              : Number(additionalInformation.usesAccessibleConvertedVan)
           }
         >
           <Stack>
-            <Radio value={'0'}>{'Yes'}</Radio>
-            <Radio value={'1'}>{'No'}</Radio>
+            <Radio value={1} isChecked={additionalInformation.usesAccessibleConvertedVan ?? false}>
+              {'Yes'}
+            </Radio>
+            <Radio value={0} isChecked={!additionalInformation.usesAccessibleConvertedVan ?? false}>
+              {'No'}
+            </Radio>
           </Stack>
-        </RadioGroup>
-      </FormControl>
+        </RadioGroupField>
+      </Box>
 
-      <FormControl as="fieldset" isRequired paddingTop="24px">
-        <FormLabel>{'Does the applicant need a wider accessible parking space?'}</FormLabel>
-        <RadioGroup
-          value={
-            data.requiresWiderParkingSpace === null
-              ? undefined
-              : data.requiresWiderParkingSpace
-              ? '0'
-              : '1'
-          }
-          onChange={value =>
-            onChange({ ...data, requiresWiderParkingSpace: value === '0' ? true : false })
-          }
-        >
-          <Stack>
-            <Radio value="0">{'Yes'}</Radio>
-            <Radio value="1">{'No'}</Radio>
-          </Stack>
-        </RadioGroup>
-      </FormControl>
+      {Number(additionalInformation.usesAccessibleConvertedVan) === 1 && (
+        <Box paddingBottom="24px">
+          <RadioGroupField
+            name="additionalInformation.accessibleConvertedVanLoadingMethod"
+            label="Please specify their loading method:"
+            required
+          >
+            <Stack>
+              <Radio value={AccessibleConvertedVanLoadingMethod.SIDE_LOADING}>
```

Comment:
```suggestion
              <Radio value="SIDE_LOADING">
```
Please use GraphQL enums (strings) instead of Prisma enums